src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -167,10 +167,11 @@ export default class ReimbursementRequestService {
    * Function to reimburse a user for their expenses.
    *
    * @param amount the amount to be reimbursed
+   * @param dateReceived the date the amount was received
    * @param submitter the person performing the reimbursement
    * @returns the created reimbursement
    */
-  static async reimburseUser(amount: number, submitter: User): Promise<Reimbursement> {
+  static async reimburseUser(amount: number, dateReceived: Date, submitter: User): Promise<Reimbursement> {
```

Comment:
this is actually a string because we're getting it from Json, you'll need to type it as a string here and make a new Date with it like this (or at least this is how we do it now, if you have a better solution, that would be lit): 
`    // make the date object but add 12 hours so that the time isn't 00:00 to avoid timezone problems`
`    const date = new Date(startDate.split('T')[0]);`
   ` date.setTime(date.getTime() + 12 * 60 * 60 * 1000);`

Reply:
`ValidationChain.toDate()`, but doesn't seem to be much documentation on this https://express-validator.github.io/docs/api/validation-chain/#todate